draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -320,18 +320,19 @@ verification key for the Issuer-signed JWT:
 
 - JWT Issuer Metadata: If the `iss` value contains an HTTPS URI, the recipient MUST
 obtain the public key using JWT Issuer Metadata as defined in (#jwt-issuer-metadata).
+- X.509 Certificates: The recipient MUST obtain the public key from the leaf X.509 certificate
+corresponding to the `x5c`, `x5c`, `x5t`, or `x5t#S256` JWT header parameters of the Issuer-signed JWT if one of them is present. In this case, the recipient MUST validate the X.509 certificate chain and the recipient MUST validate the `iss` value as follows:
+    - If the `iss` value contains a DNS name encoded as a URI using the DNS URI scheme [@RFC4501], the DNS name MUST match a `dNSName` Subject Alternative Name (SAN) [@RFC5280] entry of the leaf certificate.
+    - In all other cases, the `iss` value MUST match a `unifiedResourceIdentifier` SAN entry of the leaf certificate.
 - DID Document Resolution: If the `iss` value contains a DID [@W3C.DID], the recipient MUST retrieve
 the public key from the DID Document resolved from the DID in the `iss` value.
 In this case, if the `kid` JWT header parameter is present, the `kid` MUST be a relative or absolute
 DID URL of the DID in the `iss` value, identifying the public key.
-- X.509 Certificates: The recipient MUST obtain the public key from the leaf X.509 certificate
-defined by the `x5c`, `x5c`, or `x5t` JWT header parameters of the Issuer-signed JWT and validate the X.509
-certificate chain in the following cases:
-    - If the `iss` value contains a DNS name encoded as a URI using the DNS URI scheme [@RFC4501]. In this case, the DNS name MUST match a `dNSName` Subject Alternative Name (SAN) [@RFC5280] entry of the leaf certificate.
-    - If the `iss` value contains a URN using the URN URI scheme [@RFC2141]. In this case, the URN MUST match a `unifiedResourceName` SAN entry of the leaf certificate.
 
 Separate specifications or ecosystem regulations MAY define rules complementing the rules defined above, but such rules are out of scope of this specification. See (#ecosystem-verification-rules) for security considerations.
 
+If the public verification key for the Issuer-signed JWT could not be obtained, the recipient MUST NOT use the public key to verify the Issuer-signed JWT.
```

Comment:
This sentence reads a bit funny - almost like a tautology or proverb, "If you can't get it, you must not use it." Maybe something like one of these (some admittedly somewhat awkward):

"If the public verification key for the Issuer-signed JWT could not be obtained according to rules above, the recipient MUST reject the SD-JWT VC."
or
"A recipient MUST only use a public verification key obtained according to rules above to verify the signature of an Issuer-signed JWT."
or 
"A public verification key that was not obtained according to rules above MUST not be used to verify the Issuer-signed JWT."

Reply:
I think we should drop this requirement completely. It does not say much. We should instead do what is described in #185 and then it becomes obvious that "verification and processing" would fail if the rules cannot be successfully applied.

Reply:
Would this work for you @bc-pi ?

Reply:
I think so, yeah.

Reply:
Great, I'll update the PR accordingly